sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/WhereLogic.cs

```diff
@@ -56,7 +54,7 @@ public WhereQuery Where(string value, QueryContainsOperator containOperator, str
         /// </summary>
         /// <param name="condition"> The verbatim condition (SQL-like syntax) in string format. </param>
         /// <returns> ADT query that already contains SELECT and FROM. </returns>
-        public WhereQuery Where(string condition)
+        public WhereLogic Override(string condition)
```

Comment:
Not the biggest fan of the `Override` name ... I still like `IsTrue` better than this ...

Reply:
What does everyone else think?

Reply:
Yeah that's fair -- I didn't circle back to that one. Could we overload `IsTrue` with the override, the contains and the comparison methods?

Reply:
I don't like either, honestly.

This is just a user providing their own WHERE clause segment? Like `c.Foo = "bar"`?

Reply:
Yes. What would you suggest?

Reply:
`AddCustom(string whereSegment)`?

Reply:
it would look like :
`Select(*).From("DigitalTwins").Where().AddCustom("2 > 3") `
me no like :D you guys like?

Reply:
`Where().CustomClause()`?

Reply:
much better!